domain-get-subdomains-deep/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "route": "v1/domains/subdomains/deep/{id}",
```

Comment:
This seems fine, but what if we added a query param to `/subdomains` that was `?deep=true`

Reply:
Jk, i'm realizing that this endpoint doesn't support the same sorting